lib/constants.js

```diff
@@ -14,6 +14,14 @@ exports.LOG_ERROR = 'ERROR'
 exports.LOG_WARN = 'WARN'
 exports.LOG_INFO = 'INFO'
 exports.LOG_DEBUG = 'DEBUG'
+exports.LOG_PRIORITIES = [
```

Comment:
right, but we need to add it.
LOG_LOG maps to 'console.log' function as well as LOG_DEBUG maps to 'console.debug'
Try your pull request with console.log and browserConsoleLogOptions.level = 'debug' and you will see that it doesn't work, cause 
constants.LOG_PRIORITIES.indexOf(type)
will have -1 (not found)